.github/workflows/update-operators-cache.yml

```diff
@@ -0,0 +1,54 @@
+name: Update Operators Cache
+
+on:
+  workflow_dispatch:
+    inputs:
+      mainnet-blob-id:
+        description: 'Mainnet blob ID'
+        required: true
+        type: string
+      testnet-blob-id:
+        description: 'Testnet blob ID'
+        required: true
+        type: string
+  # schedule:
+  #   - cron: '0 11 * * 1'  # Every Monday at 11:00 UTC
+
+env:
+  MAINNET_BLOB_ID: ${{ github.event.inputs.mainnet-blob-id || '6SCk-C85lApQY_3bGwJMpVve5ASWiIhxWplg22fKHvo' }}
+  TESTNET_BLOB_ID: ${{ github.event.inputs.testnet-blob-id || 'AYTUffH740r9eHKRgICusBEw-M6DucKqIBAHctZDCC8' }}
+
+jobs:
+  update-cache:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install dependencies
+        run: cd scripts/cache-inference && npm install
```

Comment:
Remark: This currently results in the creation of a `package-lock.json` file, which is subsequently included in the PR, see [here](https://github.com/MystenLabs/walrus/pull/1988/files).